_includes/categorylinks.html

```diff
@@ -1,10 +1,12 @@
-    <div class="tablet:grid-col">
-      {% for link in include.links %}
-        {% if forloop.first == true %}
-          <h2 class = "usa-prose">{{ link.text }}</h2>
-          <a href = "{{ link.href }}">Overview</a><br />
-        {% else %}
-          <a href = "{{ link.href }}">{{ link.text }}</a><br />
-        {% endif %}
-      {% endfor %}
-    </div>
+<div class="tablet:grid-col">
+  {% for link in links %}
+    {%- if forloop.first == true -%}
+      <h2 class = "usa-prose">{{ link.title }}</h2>
+      <a href = "{{ link.url }}">Overview</a>
+    {%- else -%}
+      <p class="margin-bottom-05 margin-top-0">
+        <a href = "{{ link.url }}">{{ link.title }}</a>
```

Comment:
Getting a 404 for all of the links on the federalist preview. I think we need to add the `url` filter, as in `{{ link.url | url }}`

Reply:
Fixed in d8717cd

Reply:
Looks like this one is still broken – I think you put the url filter on the titles rather than on the `href` value of the `a` tag.

Reply:
Yikes! Should be good now: https://github.com/18F/guides/pull/64/commits/b146a1b8f8f33eeaeb9179f0a71f4c369ee848e8 but will check the Pages build

Reply:
Looks good now!